lego/apps/files/views.py

```diff
@@ -46,6 +46,21 @@ def create(self, request, *args, **kwargs):
             status=status.HTTP_201_CREATED,
         )
 
+    @decorators.action(
+        detail=True,
+        methods=["PATCH"],
+        permission_classes=[permissions.AllowAny],
+    )
+    def set_save_for_use(self, request, *args, **kwargs):
+        file = self.get_object()
+        serializer = FileSaveForUseSerializer(data=request.data)
+        if serializer.is_valid():
+            file.save_for_use = serializer.validated_data["save_for_use"]
+            file.save()
+            return Response({"Success"})
```

Comment:
Avoid using arbitrary return values here and rather just return a `HTTP_200` f.ex.